include/groonga/groonga.h

```diff
@@ -59,6 +59,7 @@ typedef __bf16 grn_bfloat16;
 #define GRN_FALSE  false
 
 typedef enum {
+  /// Success
```

Comment:
The generated document is this.
![Screenshot from 2024-08-15 23-40-43](https://github.com/user-attachments/assets/ca6fc3c0-cbaa-4db8-be21-d98eb1617fc2)